### DISCUSSION TITLE
Add .htaccess files in /js and /skin directories

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2288] [Ideas]
I would like to create a new PR but first I need to listen to some opinions. In the **/media/.htaccess** file we disable PHP code execution and do not permit execution of CGI scripts. See bellow

```php
############################################
## Disable PHP code execution

<IfModule mod_php7.c>

    php_flag engine 0

</IfModule>

<IfModule mod_php8.c>

    php_flag engine 0

</IfModule>


############################################
## Don't permit execution of CGI scripts

AddHandler cgi-script .php .pl .py .jsp .asp .htm .shtml .sh .cgi
Options -ExecCGI

```

I would like to add .htaccess files to the /js and /skin directories to do the same. These directories contain js, css, image files, but have no scripts to run.

Is this file a good idea? If so, should .htm and .shtml files be excluded from these directories too?

**Comment by fballiano** [comment]
.htm shouldn't be scripts so i think we could leave them alone, all the rest seems pretty good to me

**Comment by addison74** [comment]
This was implemented in OpenMage here https://github.com/OpenMage/magento-lts/pull/2328 and https://github.com/OpenMage/magento-lts/pull/2329.
